### DISCUSSION TITLE
Please share any ideas for new features in this discussion thread.

### DISCUSSION THREAD

**Discussion by bentleymi** [bentleymi/ta-openai-api#6] [Ideas]
We want to make the app powerful and as user friendly and easy to use as possible.  Please share any ideas you may have for improving it.  Thank you.

**Comment by bentleymi** [comment]
I'll kick it off with a request to add a "debug" feature that prints the rest of the options that were used, and change the default behavior to only contain the response.

**Comment by bentleymi** [comment]
We really could use an improved setup screen that shows each orgId that has been added to passwords.conf and allows you to add/edit/delete orgIds

**Comment by firoagni** [comment]
It would be great if you can support this amazing Splunk app with local models such as Ollama + Ilama 3.2
